pkg/networks/networks.yaml

```diff
@@ -1,15 +1,19 @@
-# Paths to vde executables. Because vde_vmnet is invoked via sudo it should be
+# Paths to socket_vmnet executables. Because socket_vmnet is invoked via sudo it should be
 # installed where only root can modify/replace it. This means also none of the
 # parent directories should be writable by the user.
 #
 # The varRun directory also must not be writable by the user because it will
-# include the vde_vmnet pid files. Those will be terminated via sudo, so replacing
+# include the socket_vmnet pid files. Those will be terminated via sudo, so replacing
 # the pid files would allow killing of arbitrary privileged processes. varRun
```

Comment:
```suggestion
# include the socket_vmnet pid file. It be terminated via sudo, so replacing
# the pid file would allow killing of arbitrary privileged processes. varRun
```

Reply:
Fixed